CONTRIBUTING.md

```diff
@@ -1,21 +1,140 @@
 # Contributing
 
-Contributions are more than welcome. Bug reports with specific reproduction steps are great. If you have a code contribution you'd like to make, open a pull request with suggested code.
+Contributions are more than welcome.
+Bug reports with specific reproduction steps are great.
+If you have a code contribution you'd like to make, open a pull request with suggested code.
 
-Note that PR's and issues are reviewed every ~2 weeks. If your PR or issue is critical in nature, please reflect that in the description so that it receives faster attention.
+Note that PR's and issues are reviewed every ~2 weeks.
+If your PR or issue is critical in nature, please reflect that in the description so that it receives faster attention.
 
-Pull requests should:
-- Clearly state their intent in the title
-- Have a description that explains the need for the changes
-- Include tests! (Make sure the recipe works and converges)
-- Not break the public API
-- Add yourself to the CONTRIBUTING file at the bottom
-- Increment the recipe version info in metadata.rb
-- Test some more!
+Parts of this documentation have been adopted from the [Chef community cookbook documentation][8].
 
-By contributing to this project you agree that you are granting New Relic a non-exclusive, non-revokable, no-cost license to use the code, algorithms, patents, and ideas in that code in our products if we so choose. You also agree the code is provided as-is and you provide no warranties as to its fitness or correctness for any purpose
+## Pull requests
 
-Copyright (c) 2017 New Relic, Inc. All rights reserved.
+* Clearly state their intent in the title
+* Have a description that explains the need for the changes
+* Include tests! (Make sure the recipe works and converges)
+* Not break the public API
+* Add yourself to the CONTRIBUTING file at the bottom
+* Increment the recipe version info in metadata.rb
+* Test some more!
+
+By contributing to this project you agree that you are granting New Relic a non-exclusive, non-revokable, no-cost license to use the code, algorithms, patents, and ideas in that code in our products if we so choose. You also agree the code is provided as-is and you provide no warranties as to its fitness or correctness for any purpose.
+
+## Testing
+
+### Installing dependencies
+
+The gem `bundler` must be installed for dependency management.
```

Comment:
Instead of requiring bundler, we should require ChefDK and assume the user has a recent version of that.